app/src/pages/lloydGeorgeRecordPage/LloydGeorgeRecordPage.tsx

```diff
@@ -122,6 +124,7 @@ function LloydGeorgeRecordPage() {
                             numberOfFiles={numberOfFiles}
                             refreshRecord={refreshRecord}
                             cloudFrontUrl={cloudFrontUrl}
+                            showMenu={showMenu}
                         />
```

Comment:
Im not seeing any new unit tests for the LloydGeorgeRecordPage to reflect this new prop. Need tests to check what happens when this is toggled

Reply:
there is a test for `Does not render the MenuCard if showMenu is false` in RecordMenuCard, do you think we need to tests the LloydGeorgeRecordPage ?

Reply:
generally speaking we should but as there is already so much bloat in these tests we are probably fine